source/features/highlight-non-default-base-branch.tsx

```diff
@@ -18,16 +18,12 @@ function isClosed(prLink: HTMLElement): boolean {
 }
 
 function buildQuery(issueIds: string[]): string {
-	return `
-		repository() {
-			${issueIds.map(id => `
+	return issueIds.map(id => `
```

Comment:
This is the nice side effect: no more awkward `array map inside template literal`